code/lib/cli/src/project_types.ts

```diff
@@ -254,6 +255,14 @@ export const supportedTemplates: TemplateConfiguration[] = [
       return dependencies.every(Boolean);
     },
   },
+  {
+    // TODO: This only works because it is before the SVELTE template. could be more explicit
+    preset: ProjectType.SVELTE_KIT,
+    dependencies: ['@sveltejs/kit'],
+    matcherFunction: ({ dependencies }) => {
+      return dependencies.every(Boolean);
+    },
+  },
   {
     preset: ProjectType.SVELTE,
     dependencies: ['svelte'],
```

Comment:
A stricter solution would be for this `ProjectType.SVELTE`'s matcher function to not match if it sees the `@sveltejs/kit` dependency, but I can't figure out how this whole code works, so I can't make it work.